packages/postcss-svgo/package.json

```diff
@@ -28,16 +28,16 @@
   "repository": "cssnano/cssnano",
   "dependencies": {
     "postcss-value-parser": "^4.2.0",
-    "svgo": "^2.7.0"
+    "svgo": "^3.0.1"
   },
   "bugs": {
     "url": "https://github.com/cssnano/cssnano/issues"
   },
   "engines": {
-    "node": "^10 || ^12 || >=14.0"
+    "node": "^14 || ^16 || >= 18"
   },
   "devDependencies": {
-    "@types/svgo": "^2.6.2",
+    "@types/csso": "^5.0.0",
```

Comment:
This can be removed now as they are no longer needed since v3.0.2 of SVGO.

Reply:
Removed :)